src/Entity/Journey.php

```diff
@@ -43,4 +56,96 @@ class Journey
      * @ORM\OneToMany(targetEntity="App\Entity\Message", mappedBy="journey", cascade={"all"})
      */
     private $messages;
+
+    public function __construct()
+    {
+        $this->steps = new ArrayCollection();
+        $this->startAt = new \DateTime();
```

Comment:
`@Assert\GreaterThan("now")`and `$this->startAt = new \DateTime();` it's a trap ! preset data with no valid data is not very nice :trollface:

Reply:
👍